### PR TITLE
Add SIG Stack Guidance

### PR DESCRIPTION
**Pull request by mayaCostantini**
## This introduces a breaking change

- No

## This Pull Request implements

Add SIG Stack Guidance description and details.

### CODE REVIEW

**Comment by mayaCostantini** [comment]
@fridex PTAL if this needs any additions or changes :+1:

**Review by fridex — COMMENTED** [review]
Changes look good to me (however, I'm not familiar with these YAML files) 👍🏻 

/lgtm

File: community/sig-stack-guidance/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+
+for Thoth we use `podman run --rm -e WHAT -e GO111MODULE=on -e GOPROXY -v $(pwd):/go/src/app:Z golang:1.12 make -C /go/src/app generate`
+
+--->
+
+# Stack Guidance Special Interest Group
+
+Work on recommending the most effective, performant and secure software stack for user applications.
```
Comment (goern) [inline community/sig-stack-guidance/README.md:15]: I'd like to see the topic of 'actively creating prescriptions and security related information' somehow covered. @mayaCostantini @fridex

**Review by goern — CHANGES_REQUESTED** [review]

File: community/sig-stack-guidance/README.md (hunk 1 above)
Comment (mayaCostantini) [reply community/sig-stack-guidance/README.md]: Thanks, added in https://github.com/thoth-station/core/pull/375/commits/4ed056cb80b5d0a80935ba187cdc6a02a7466a41 :+1:

File: community/sigs.yaml [hunk 2]
```diff
@@ -1,192 +1,240 @@
...
+    day: Thursday
+    time: "15:00"
+    tz: UTC (Coordinated Universal Time)
+    frequency: weekly
+    url: https://meet.google.com/kxd-axiz-tym
+  contact:
+    liaison:
+      github: goern
+      name: Christoph Görn
+  subprojects:
```
Comment (codificat) [inline community/sigs.yaml:184]: I think there are more repos to be "owned' by this SIG (and therefore should be listed here)...
I believe that the intention was for the SIG to also cover the Knowledge graph, right? That would include e.g. `storages` and other related repos

File: community/sigs.yaml [hunk 3]
```diff
@@ -1,192 +1,240 @@
...
+  - description: Let's talk Thoth Tech
+    day: Thursday
+    time: "15:00"
+    tz: UTC (Coordinated Universal Time)
+    frequency: weekly
+    url: https://meet.google.com/kxd-axiz-tym
+  contact:
+    liaison:
+      github: goern
+      name: Christoph Görn
```
Comment (codificat) [inline community/sigs.yaml:183]: Wouldn't that be @fridex for this SIG?

Comment (mayaCostantini) [reply community/sigs.yaml:183]: I saw that @goern was the `contact` for all other SIGs, so I assumed this was not something that should be changed SIG-wise

File: community/sigs.yaml (hunk 2 above)
Comment (mayaCostantini) [reply community/sigs.yaml:184]: I think this is something we should discuss further with @fridex, would it be possible to add other repos later?

File: community/sigs.yaml [hunk 4]
```diff
@@ -1,192 +1,240 @@
...
-      - name: s2i-thoth
-        owners:
-          - https://raw.githubusercontent.com/thoth-station/s2i-thoth/master/OWNERS
-      - name: thamos
-        owners:
-          - https://raw.githubusercontent.com/thoth-station/thamos/master/OWNERS
-      - name: user-api
-        owners:
-          - https://raw.githubusercontent.com/thoth-station/user-api/master/OWNERS
+  charter_link: charter.md
```
[the remark this answers is not in the document]
Comment (mayaCostantini) [reply community/sigs.yaml:159]: ThanksI will add it, I thought this was auto-generated. I will also add the `OWNERS` file which I think is missing too.

**Comment by codificat** [comment]
Thanks!

A few improvements are needed in the charter document, but not only on this one but across all SIGs, so let's address that in a separate PR.

/approve

**Pull request merged**